tests/contract_test.rs

```diff
@@ -15,6 +15,7 @@ async fn init_server(script: String) -> Result<String, Box<dyn std::error::Error
 async fn main() {
     env_logger::builder().try_init().unwrap();
     PubNubWorld::cucumber()
+        .max_concurrent_scenarios(1)
```

Comment:
I know that this is added because of Tomato's behaviour but imo some kind of comment why it cannot be run in concurrent style would be great.